Make the documentation correct for concurrent default

jkburges opened the pull request:
This corresponds to the default set at https://github.com/JoeDog/siege/blob/master/doc/siegerc.in#L267

modsaid reviewed (approved):
makes sense... 

suggestion: do we need to clarify where that the default can change in the config files as well?

jkburges commented:
> suggestion: do we need to clarify where that the default can change in the config files as well?

I don't understand what you mean here, sorry.

modsaid commented:
@jkburges   sorry about that, let me elaborate.. 

The current default is 25 (in the configuration file).. the help message indicates default of 10 which is incorrect.
Changing to 25 should be correct as a default.

However, if the help message changed to 25,   and the the config file was changed (by the user after installation or by any other user with access to the system, to 20 for example)...   the cli usage would still indicate 25 which is now incorrect

One suggestion can be looking up the value in the config file and populate it as the default in the help message...  but that's a bit of an overkill

Another suggestion, which i meant above, is: Perhaps in the help message itself, we can state that the default might be overwritten in the help file.

The whole point is not deceiving the end user by giving incorrect info

jkburges committed:
Make the documentation correct for concurrent default

This corresponds to the default set at https://github.com/JoeDog/siege/blob/master/doc/siegerc.in#L267

jkburges commented:
@modsaid I understand, thanks.

Have added a note about that - and I had a conflict and a bunch of unintentional whitespace changes, so I've force pushed a new commit.

modsaid commented:
Thanks,  I guess it is up to @JoeDog  now..
